airbyte-lib/tests/integration_tests/test_source_test_fixture.py

```diff
@@ -682,11 +707,15 @@ def test_sync_to_postgres(new_pg_cache_config: PostgresCacheConfig, expected_tes
 
     assert result.processed_records == 3
     for stream_name, expected_data in expected_test_stream_data.items():
-        pd.testing.assert_frame_equal(
-            result[stream_name].to_pandas(),
-            pd.DataFrame(expected_data),
-            check_dtype=False,
-        )
+        if len(cache[stream_name]) > 0:
+            pd.testing.assert_frame_equal(
+                result[stream_name].to_pandas(),
+                pd.DataFrame(expected_data),
+                check_dtype=False,
+            )
```

Comment:
For my info, does `assert_frame_equal()` not work with an empty dataset?

Reply:
nope, trying to run .to_pandas() on an empty dataset fails, not meant to work apparently.

Reply:
Makes sense! Thanks for confirming. 👍